projects/testing/cypress/snapshot/command.ts

```diff
@@ -1,6 +1,65 @@
+/// <reference types="cypress" />
+
 import {Options} from 'cypress-image-snapshot';
-import {addMatchImageSnapshotCommand} from 'cypress-image-snapshot/command';
+import {matchImageSnapshotCommand} from 'cypress-image-snapshot/command';
+
+import {tuiWaitAllImgInside} from '../commands/wait-all-img-inside.command';
+
+declare module 'cypress-image-snapshot/command' {
+    function matchImageSnapshotCommand(options: Options): (...args: unknown[]) => void;
+}
+
+interface TuiSnapshotCommandOptions {
+    waitAllImages?: boolean;
+}
+
+declare global {
+    namespace Cypress {
+        interface Chainable {
+            matchImageSnapshot(
+                name: string,
+                options?: Options & TuiSnapshotCommandOptions,
+            ): void;
+        }
+    }
+}
+
+export function tuiAddMatchImageSnapshotCommand(
+    options: Options & TuiSnapshotCommandOptions,
+): void {
+    const matchSnapshotFn = matchImageSnapshotCommand(options);
+
+    Cypress.Commands.add(
+        `matchImageSnapshot`,
+        {prevSubject: [`optional`, `element`, `window`, `document`]},
+        (
+            prevSubject,
+            nameOrOptions: string | (Options & TuiSnapshotCommandOptions),
+            options?: Options & TuiSnapshotCommandOptions,
+        ) => {
+            const name = typeof nameOrOptions === `string` ? nameOrOptions : undefined;
+
+            tuiWaitAllImgInside(prevSubject, options?.waitAllImages ?? true);
+
+            matchSnapshotFn(prevSubject, makeScreenshotName(name), options);
+        },
+    );
+}
+
+const globalScreenshotsInfo = new Map<string, number>();
+
+function makeScreenshotName(name?: string): string {
+    const screenshot =
+        name ??
+        Cypress.currentTest.titlePath
+            .join(`-`)
+            .replace(/\s/g, `.`)
+            .replace(/'/g, ``)
```

Comment:
What about other quotes ?

For example,
https://github.com/Tinkoff/taiga-ui/blob/d96b6d1ace5b5c3720224b489d79105161a743af/projects/demo-integrations/cypress/tests/addon-table/table-pagination/table-pagination.spec.ts#L8

Reply:
fixed